### PR TITLE
MBS-7863 Add a post about open source

### PR DESCRIPTION
**Pull request by eugene-krivobokov**
These are just copies of already published articles:

- [habrahabr](https://habr.com/ru/company/avito/blog/496036/) 
- [Medium](https://medium.com/avitotech/open-source-ci-cd-and-test-infrastructure-for-android-95d056619787)

### CODE REVIEW

File: docs/content/posts/lint-worker-api-workaround.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 date: 2020-02-20
 title: Android lint and Gradle worker API workaround
 tags: [gradle, agp, lint]
-weight: 10
```
Comment (eugene-krivobokov) [inline docs/content/posts/lint-worker-api-workaround.md:5]: Removed weight to sort by automatically by date

File: docs/content/posts/open-source-introduction.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
+---
+date: 2019-12-26
```
Comment (eugene-krivobokov) [inline docs/content/posts/open-source-introduction.md:2]: The date of the [initial commit](https://github.com/avito-tech/avito-android/commit/bc52d94ffc4121e4a901112fb77ecf3660d7f221)

File: docs/content/posts/open-source-introduction.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+title: Open source
```
Comment (eugene-krivobokov) [inline docs/content/posts/open-source-introduction.md]: Can't use ":" symbol or escape it in a page title.

How about "Open source for CI/CD and test infrastructure for Android"?

File: docs/content/posts/open-source-introduction.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+- Emulators.
+ 
+Let’s discuss some of the most important libraries.
+
+### Test runner
+
+This is a Gradle plugin to run instrumentation tests. 
+The closest analog is [Marathon](https://github.com/Malinskiy/marathon), but our plugin runs only under Android.
+ 
+[Test runner:]({{< ref "/docs/test/Runner.md" >}})
```
Comment (eugene-krivobokov) [inline docs/content/posts/open-source-introduction.md:146]: Internal links are for good. They are always correct.

File: docs/content/posts/open-source-introduction.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+In this review, we will explain why we decided to move into open source, present the central libraries of the project, and suggest whom to contact with any questions. 
+We will analyze in detail the individual libraries, Gradle plugins, and our development approaches in future posts.
+ 
+![ijhasr2dv95azaem9mgilu7e1kw](https://user-images.githubusercontent.com/1104540/80410956-0cf44200-88d4-11ea-879d-cd9b6acd29af.png)
+
+## Who we are and what we do
+
+We are developing solutions for Android as part of the Speed platform team. There are four of us:
+
+{{< columns >}}
```
Comment (eugene-krivobokov) [inline docs/content/posts/open-source-introduction.md:22]: Sometimes the layout is not good, but I think this is ok for now.

File: docs/content/posts/open-source-introduction.md (hunk 3 above)
Comment (int02h) [reply docs/content/posts/open-source-introduction.md]: "Open source for CI/CD and test infrastructure for Android" sounds great

File: docs/content/posts/open-source-introduction.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+Senior engineer\
+[eugene_kr](https://twitter.com/eugene_kr)
+
+![](https://user-images.githubusercontent.com/1104540/80413105-9a856100-88d7-11ea-9e07-b5a5e48882cf.png)
+
+<--->
+
+Daniil Popov\
+Senior engineer\
+[Int02h](https://twitter.com/Int02h)
```
Comment (int02h) [inline docs/content/posts/open-source-introduction.md]: I don't think Twitter links are case-sensitive but I'd change this to `int02h` with the first letter in lower-case.

Comment (eugene-krivobokov) [reply docs/content/posts/open-source-introduction.md]: They are case-insensitive. Ok, I'll change

Comment (eugene-krivobokov) [inline docs/content/posts/open-source-introduction.md]:
```suggestion
[Int02h](https://twitter.com/int02h)
```

File: docs/content/posts/open-source-introduction.md (hunk 3 above)
Comment (eugene-krivobokov) [reply docs/content/posts/open-source-introduction.md]:
```suggestion
title: Open source for CI/CD and test infrastructure for Android
```

**Comment by eugene-krivobokov** [comment]
Ready for review

**Comment by eugene-krivobokov** [comment]
Ping

**Review by int02h — APPROVED** [review]

**Pull request merged**